Classes/Util/PBOpenFiles.h

```diff
@@ -8,9 +8,11 @@
 
 #import <Foundation/Foundation.h>
 
+@class PBChangedFile;
+
 @interface PBOpenFiles : NSObject
 
-+ (void)showInFinderAction:(id)sender with:(NSURL *)workingDirectoryURL;
-+ (void)openFilesAction:(id)sender with:(NSURL *)workingDirectoryURL;
++ (void)openFiles:(NSArray<PBChangedFile *> *)selectedFiles with:(NSURL *)workingDirectoryURL;
```

Comment:
My line of thinking was the other way round … a) the code does only work for PBChangedFiles and b) it looks like a Utility to handle opening files.

I can see how it makes sense to add the ability to open other objects here when it’s needed. However, seeing this as a utility for opening files, I don‘t think it should receive UI-Elements as an input and expect them to represent specific objects. My idea was to extract the data from the UI at the controller layer and let this class handle opening the files.